client/my-sites/stats/stats-module/index.jsx

```diff
@@ -154,56 +154,61 @@ class StatsModule extends Component {
 			'is-refreshing': requesting && ! isLoading,
 		} );
 
-		const shouldHideOldModule = isEnabled( 'stats/new-stats-module-component' ) && path === 'posts';
+		const shouldShowNewModule =
+			isEnabled( 'stats/new-stats-module-component' ) &&
+			! summary &&
+			( path === 'posts' || path === 'videoplays' );
 
 		return (
 			<>
 				{ siteId && statType && (
 					<QuerySiteStats statType={ statType } siteId={ siteId } query={ query } />
 				) }
-				{ isEnabled( 'stats/new-stats-module-component' ) && ! summary && path === 'posts' && (
-					<StatsListPosts
-						moduleName={ path }
-						data={ data }
-						useShortLabel={ useShortLabel }
-						title={ this.getModuleLabel() }
-						emptyMessage={ moduleStrings.empty }
-						showMore={
-							this.props.showSummaryLink && displaySummaryLink
-								? {
-										url: this.getHref(),
-										label: this.props.translate( 'View all', {
-											context: 'Stats: Button label to expand a panel',
-										} ),
-								  }
-								: undefined
-						}
-						titleURL={ this.getHref() }
-						error={ hasError && <ErrorPanel /> }
-						loader={ isLoading && <StatsModulePlaceholder isLoading={ isLoading } /> }
-					>
-						{ this.props.children }
-					</StatsListPosts>
+				{ shouldShowNewModule && (
+					<>
+						<StatsListCard
```

Comment:
Do we intend to nest more than `StatsListCard` in this fragment in the future?

Reply:
Good catch. I'll remove the fragment.